app/javascript/src/components/Rondel.vue

```diff
@@ -47,31 +47,19 @@
       >
         <div v-if="game.baseGame === 'imperial'">
           <div>
-            <b>
-              Current Tax Chart Position:
-            </b> {{ game.nations.get(game.currentNation).taxChartPosition }}
+            <b> Tax Chart Position </b> will go from <b>{{ game.nations.get(game.currentNation).taxChartPosition }}</b> to <b>{{ nextTaxChartPosition }}</b>
           </div>
           <div>
-            <b>Next Tax Chart Position:</b> {{ nextTaxChartPosition }}
+            <b> {{ game.currentPlayerName }} </b> would receive {{ nextTaxChartPosition - game.nations.get(game.currentNation).taxChartPosition }}m
           </div>
           <div>
-            <b>
-              {{ game.currentPlayerName }}
-            </b> would receive
-            {{ nextTaxChartPosition - game.nations.get(game.currentNation).taxChartPosition }}m
+            <b> {{ stringify(game.currentNation.value) }} </b> would receive {{ nextTaxChartPosition - game.unitCount(game.currentNation) }}m
           </div>
           <div>
-            <b>
-              {{ stringify(game.currentNation.value) }}
-            </b> would receive {{ nextTaxChartPosition - game.unitCount(game.currentNation) }}m
-          </div>
-          <div>
-            <b>
-              {{ stringify(game.currentNation.value) }}
-            </b>'s power points would be {{ nextTaxationPowerPoints }}
+            <b> {{ stringify(game.currentNation.value) }} </b>'s power points would be {{ nextTaxationPowerPoints }}
           </div>
         </div>
-        <div v-else-if="game.baseGame === 'imperial2030'">
+        <div v-else-if="game.baseGame === 'imperial2030' || game.baseGame === 'imperialAsia'">
```

Comment:
Good catch. I _think_ for `imperialAsia` we're generally defaulting to 2030 rules for taxation so this will likely be accurate. But tbh I'm paying nearly zero attention to `imperialAsia` so who knows! 🤷